matdeeplearn/preprocessor/processor.py

```diff
@@ -314,6 +328,23 @@ def get_data_list(self, dict_structures, y):
         logging.info("Generating edge features...")
         generate_edge_features(data_list, self.edge_steps, self.r, device=self.device)
 
+        # compile non-otf transforms
+        logging.debug("Applying transforms.")
+        transforms_list = []
+        for transform in self.transforms:
+            if not transform["otf"]:
```

Comment:
Same here, let's do:
```suggestion
            if not transform.get("otf", False):
```